### PR TITLE
Bundle contest style and scoreboard Uriel endpoints

### LINKED ISSUE #111: Bundle Contest Scoreboard
Create bundle contest scoreboard page.

### PR DESCRIPTION
**Pull request by nathanchrs**
Uriel endpoints for #111.

Resolve #111

### CODE REVIEW

File: judgels-backends/uriel/uriel-api/src/main/java/judgels/uriel/api/contest/scoreboard/ScoreboardState.java [hunk 1]
```diff
@@ -11,6 +11,7 @@
 public interface ScoreboardState {
     List<String> getProblemJids();
     List<String> getProblemAliases();
+    List<Optional<Integer>> getProblemItems();
```
Comment (fushar) [inline judgels-backends/uriel/uriel-api/src/main/java/judgels/uriel/api/contest/scoreboard/ScoreboardState.java]: So as discussed, please remove this

**Review by fushar — CHANGES_REQUESTED** [review]

File: judgels-backends/uriel/uriel-app/src/test/java/judgels/uriel/contest/scoreboard/gcj/GcjScoreboardProcessorTests.java [hunk 2]
```diff
@@ -105,6 +105,7 @@ void show_only_contestant() throws JsonProcessingException {
                     styleModuleConfig,
                     contestantStartTimesMap,
                     submissions,
+                    new ArrayList<>(),
```
Comment (fushar) [inline judgels-backends/uriel/uriel-app/src/test/java/judgels/uriel/contest/scoreboard/gcj/GcjScoreboardProcessorTests.java]:
```suggestion
                    ImmutableList.of(),
```

(all changes)

File: judgels-backends/uriel/uriel-api/src/main/java/judgels/uriel/api/contest/scoreboard/ScoreboardState.java (hunk 1 above)
Comment (nathanchrs) [reply judgels-backends/uriel/uriel-api/src/main/java/judgels/uriel/api/contest/scoreboard/ScoreboardState.java]: Removed

File: judgels-backends/uriel/uriel-app/src/test/java/judgels/uriel/contest/scoreboard/gcj/GcjScoreboardProcessorTests.java (hunk 2 above)
Comment (nathanchrs) [reply judgels-backends/uriel/uriel-app/src/test/java/judgels/uriel/contest/scoreboard/gcj/GcjScoreboardProcessorTests.java]: Replaced

File: judgels-backends/uriel/uriel-app/src/main/java/judgels/uriel/contest/scoreboard/bundle/BundleScoreboardProcessor.java [hunk 3]
```diff
@@ -0,0 +1,178 @@
...
+import judgels.uriel.api.contest.module.BundleStyleModuleConfig;
+import judgels.uriel.api.contest.module.StyleModuleConfig;
+import judgels.uriel.api.contest.scoreboard.BundleScoreboard;
+import judgels.uriel.api.contest.scoreboard.BundleScoreboard.BundleScoreboardContent;
+import judgels.uriel.api.contest.scoreboard.BundleScoreboard.BundleScoreboardEntry;
+import judgels.uriel.api.contest.scoreboard.Scoreboard;
+import judgels.uriel.api.contest.scoreboard.ScoreboardState;
+import judgels.uriel.contest.scoreboard.ScoreboardProcessor;
+
+public class BundleScoreboardProcessor implements ScoreboardProcessor {
```
Comment (fushar) [inline judgels-backends/uriel/uriel-app/src/main/java/judgels/uriel/contest/scoreboard/bundle/BundleScoreboardProcessor.java:26]: Tests?

**Review by fushar — CHANGES_REQUESTED** [review]

**Review by fushar — APPROVED** [review]

File: judgels-backends/uriel/uriel-app/src/main/java/judgels/uriel/contest/scoreboard/bundle/BundleScoreboardProcessor.java (hunk 3 above)
Comment (nathanchrs) [reply judgels-backends/uriel/uriel-app/src/main/java/judgels/uriel/contest/scoreboard/bundle/BundleScoreboardProcessor.java:26]: Will be added in another PR.

**Pull request merged**